### ISSUE TITLE
Pypi package name

### ISSUE THREAD

**Issue by karolzak** [Azure-Samples/cognitive-service-vision-model-customization-python-samples#16]
Hi team!
Just out of curiosity, is it necessary to have this super long name for the python package you are building?
Couldn't you simplify it to something more user friendly, such as `uvs-python-sdk`?

**Comment by pj-ms** [comment]
Hi, Karol,

As the readme mentioned, this isn't a sdk repo, but a sample code repo.

UVS is not a meaningful name in public space either.

**Comment by karolzak** [comment]
Right, I understand it, but since you published this repo as a [package on pypi](https://pypi.org/project/cognitive-service-vision-model-customization-python-samples/) and I assume you want people to use it (even if it's not any official SDK, etc.) then I think it's worth to consider having more user friendly name (at least for the package).
Right now imports in python code look quite enormous:
```python
from cognitive_service_vision_model_customization_python_samples import abc
```
